test/webapi/ows/stac/test_controllers.py

```diff
@@ -174,7 +184,7 @@ def test_get_datasets_collection_item(self):
             "demo-1w",
         )
         self.assertIsInstance(result, dict)
-        self.assertCountEqual(self.read_json("stac-datacubes-item.json"), result)
+        assert_equal_deep_dict(self, self.read_json("stac-datacubes-item.json"), result)
```

Comment:
Previously, it was only tested that the keys in the first layer of the dictionaries are equal. Now I test the full content of the dictionaries.

Reply:
I was wondering why you define `assert_equal_deep_dict` rather than just using `self.assertEqual()`, and eventually realized that you presumably need something that's insensitive to the order of iterables within the dictionaries. I suggest adding a short docstring to `assert_equal_deep_dict` to explain this.

Reply:
> I was wondering why you define `assert_equal_deep_dict` rather than just using `self.assertEqual()`, and eventually realized that you presumably need something that's insensitive to the order of iterables within the dictionaries. I suggest adding a short docstring to `assert_equal_deep_dict` to explain this.

done